src/mmw/js/src/compare/views.js

```diff
@@ -46,6 +48,18 @@ var CompareWindow2 = Marionette.LayoutView.extend({
         sectionsRegion: '.compare-sections',
     },
 
+    initialize: function() {
+        var self = this;
+        this.model.get('scenarios').forEach(function(scenario) {
+            scenario.get('results').on('change', function(s) {
+                if (s.attributes.name === 'runoff' && !s.attributes.polling) {
```

Comment:
We should still use the backbone syntax: `s.get('name')` and `s.get('polling')`. We also may want to refer to it as `r`, for "result"